components/Form.tsx

```
@@ -0,0 +1,56 @@
+import { joiResolver } from '@hookform/resolvers/joi'
+import React, { FormEvent, ReactNode } from 'react'
+import { FormProvider, SubmitErrorHandler, useForm } from 'react-hook-form'
+
+interface IForm {
+  id?: string
+  onSubmit: (values: any) => any
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  onError?: SubmitErrorHandler<Object>
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)